src/test/resources/pages/adminActivityLogPage.html

```diff
@@ -671,11 +552,11 @@ <h4 class="text-center">
 
 
 
-    <div class="alert alert-warning" id="statusMessage">
+    <div class="alert alert-info" id="statusMessage">
           Total Logs gone through in last search: {*}<br /><input type="hidden" value="false" id="ifShowAll" /><input type="hidden" value="false" id="ifShowTestData" /></div>
 
                 
-            {*}
+            <br />
```

Comment:
Oh right, I just checked again, it should be the `<br>` above. But since the wildcard cannot represent an empty string, it should be `Total Logs gone through in last search: {*}` only (the wildcard here now will contain both the `<br>` and the "Older Logs" when it is available.